src/index.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import { AuthContextProvider } from './context/authContext';
 import { ServiceWorkerProvider } from './context/serviceWorkerContext';
 
 ReactDOM.render(
   <React.StrictMode>
     <ServiceWorkerProvider>
-      <App />
+      <AuthContextProvider>
+        <App />
+      </AuthContextProvider>
```

Comment:
thought on moving this provider to `App.jsx`?

Reply:
So you mean wrapping everything inside `App.jsx` with `AuthContextProvider` rather than here?

Reply:
yeah